kitsune/wiki/tests/test_parser.py

```diff
@@ -297,7 +296,7 @@ def test_build_template_params_named_anonymous_numbered(self):
         eq_({'1': 'a', 'hi': 'test', '3': 'z'}, _btp(['hi=test', 'a', '3=z']))
 
     def test_unapproved_template(self):
-        document(title='Template:new').save()
+        TemplateDocumentFactory(title='Template:new')
```

Comment:
And I've come full circle. I tried to make the factory always ensure there was the template prefix in place, but I can't figure out how to do that and also have a sequence-driven default value. I'd love to try and figure that out, but for now I'll push it off.